src/jit/importer.cpp

```diff
@@ -6057,6 +6057,11 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
                 // In future, it may be better to just create the right tree here instead of folding it later.
                 op1 = gtNewFieldRef(lclTyp, pResolvedToken->hField);
 
+                if (pFieldInfo->fieldFlags & CORINFO_FLG_FIELD_INITCLASS)
+                {
+                    op1->gtFlags |= GTF_FLD_INITCLASS;
```

Comment:
I searched for other appearances of `gtNewFieldRef` (and the `GenTreeField` constructor) and didn't see any others that should need this.  Instance fields are not triggers for static initialization (the rules are in [ecma 335](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwi65JmEjeTTAhUO3WMKHR1yDbYQFggiMAA&url=https%3A%2F%2Fwww.ecma-international.org%2Fpublications%2Fstandards%2FEcma-335.htm&usg=AFQjCNGmzI4xDYNpbHE274KyfnbDUQPD_g) I.8.9.5).  Address of static gets imported as an `IconHandleNode` (just a few lines down), not a FT_FIELD node.